docs/issue_template.md

```diff
@@ -0,0 +1,38 @@
+<!-- This form is for bug reports and feature requests ONLY! -->
+<!-- Thanks for filing an issue! Before hitting the button, please answer these questions.-->
+
+## Is this a BUG REPORT or FEATURE REQUEST?
+ 
+ Choose one: BUG REPORT or FEATURE REQUEST
+
+<!--
+If this is a BUG REPORT, please:
+  - Fill in as much of the template below as you can.  If you leave out information, we can't help you as well.
+
+If this is a FEATURE REQUEST, please:
+  - Describe *in detail* the feature/behavior/change you'd like to see.
+
+In both cases, be ready for followup questions, and please respond in a timely
+manner.  If we can't reproduce a bug or think a feature already exists, we
+might close your issue.  If we're wrong, PLEASE feel free to reopen it and
+explain why.
+-->
+
+
+
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+
+**Anything else we need to know?**:
+
+**Environment**:
+- Maya version (use `maya version`):
```

Comment:
Replace the maya related command with kubectl commands like for the template used in openebs/openebs

Reply:
I think, these will be more specific to maya (and other maya-component), so it will be good to know that what version of maya and component they are running(in case - if they are running some different version altogether that will help us to debug).